README.md

```diff
@@ -66,6 +66,48 @@ build:mycluster-ubuntu16-04 --jobs=64
 EOF
 ```
 
+### Platform properties
+
+For clients to be able to use buildbarn, clients need to send actions with platform properties that
+match the platform properties that buildbarn workers register themselves with.
+
+For Bazel clients, platform properties can be set in the `.bazelrc` or as command line parameters:
+
+```
+--remote_default_exec_properties=OSFamily=Linux
+--remote_default_exec_properties=container-image=docker://marketplace.gcr.io/google/rbe-ubuntu16-04@sha256:b516a2d69537cb40a7c6a7d92d0008abb29fba8725243772bdaf2c83f1be2272
+```
+
+Buildbarn workers are configured with its associated `.jsonnet` configuration file.
+
+```
+{
+  # See worker-ubuntu16-04.jsonnet for the full worker configuration.
+  buildDirectories: [{
+    native: {
+      buildDirectoryPath: '/worker/build',
+      cacheDirectoryPath: '/worker/cache',
+      maximumCacheFileCount: 10000,
+      maximumCacheSizeBytes: 1024 * 1024 * 1024,
+      cacheReplacementPolicy: 'LEAST_RECENTLY_USED',
+    },
+    runners: [{
+      endpoint: { address: 'unix:///worker/runner' },
+      concurrency: 8,
+      platform: {
+        properties: [
+          { name: 'OSFamily', value: 'Linux' },
+          { name: 'container-image', value: 'docker://marketplace.gcr.io/google/rbe-ubuntu16-04@sha256:b516a2d69537cb40a7c6a7d92d0008abb29fba8725243772bdaf2c83f1be2272' },
+        ],
+      },
+      defaultExecutionTimeout: '1800s',
+      maximumExecutionTimeout: '3600s',
+      workerId: {},
+    }],
+  }],
+}
+```
+
 Now try a build:
```

Comment:
This "Now try a build" sentence looks a bit lost with the previous subsection added.